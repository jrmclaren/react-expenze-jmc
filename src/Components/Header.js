import React from 'react';
import { NavLink } from 'react-router-dom';
const Header = () => (
  <header>
			<h1> Expenszes </h1>
			<NavLink to='/' activeClassName={"is-active"} exact>Home</NavLink>
			<br />
			<NavLink to='/create' activeClassName={"is-active"} >New Expense</NavLink>
			<br />
			
			<div>Â </div>
		</header>
);
export default Header;

