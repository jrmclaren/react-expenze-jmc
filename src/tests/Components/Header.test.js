import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Header from '../../Components/Header';

test(' should render header correctly', () => {
  const wrapper = shallow(<Header />);
  expect(wrapper).toMatchSnapshot();
//expect(wrapper.find('h1').text()).toBe(' Expenszes ');
// const renderer = new ReactShallowRenderer();
// renderer.render(<Header />);
// expect(renderer.getRenderOutput()).toMatchSnapshot();
});