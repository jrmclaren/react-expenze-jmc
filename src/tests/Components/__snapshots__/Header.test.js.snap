// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` should render header correctly 1`] = `
<header>
  <h1>
     Expenszes 
  </h1>
  <NavLink
    activeClassName="is-active"
    ariaCurrent="true"
    exact={true}
    to="/"
  >
    Home
  </NavLink>
  <br />
  <NavLink
    activeClassName="is-active"
    ariaCurrent="true"
    to="/create"
  >
    New Expense
  </NavLink>
  <br />
  <div>
    Â 
  </div>
</header>
`;
